PSKoans/Koans/IntroTraining/AboutBinary.koans.ps1

```diff
@@ -0,0 +1,180 @@
+using module PSKoans
+[Koan(Position = 109)]
+param()
+<#
+    Binary
+
+    Binary is a base 2 number system, and only uses two numbers: 0 and 1.
+
+    Binary is the basis of all computing and computational storage. It's simple & straightforward,
+    and directly represents the underlying hardware states, which can only be on (1) or off (0).
+
+    It's useful to know binary when working with PowerShell to understand how integers are represented,
+    which is our next Koan. It's also extremely useful knowledge if you plan on delving deeper into computing;
+    the knowledge is applicable to networking, programming, data science, databases, and essentially every
+    other area of computing.
+
+    What is represents is a base 2 number. 
+
+    For example, the below binary represents a "byte" of data, 
+    which contains 8 "bits". Each bit being either 1 or 0 as mentioned earlier.
+
+    So:
+
+     2^7 | 2^6 | 2^5 | 2^4 | 2^3 | 2^2 | 2^1 | 2^0
+     128 |  64 |  32 |  16 |   8 |   4 |   2 |   1 
+    -----+-----+-----+-----+-----+-----+-----+-----
+       0     0     0     0     0     0     0     1 
+    
+    Or:
+
+    00000001
+
+    Is actually 1
+
+    And 00000011
+
+    Is three. As both 1 and 2 are on, and 1 + 2 is equal to 3.
+
+#>
+
+Describe "Binary conversion" {
+
+    It "Bit conversion" {
+    <#
+        When converting from bit to a boolean try and guess what each would be
+    #>
+
+    # What would 0 be if converted to be boolean?
+    # Replace __ with either $true or $false
+    $____ -as [int] | Should -Be 0
+
+    # What would 1 be if converted to be boolean?
+    # Replace __ with either $true or $false
+    $__ -as [int] | should -Be 1
+
+    }
+
+    It "Nibble to Int conversion" {
+        <#
+            Convert the following  nibbles into full numbers
+        #>
+        
+        # Replace __ with the decimal value of 1111
+        # E.G. __ becomes 1234
```

Comment:
I'd generally consider a comment like this unnecessary, hm. 🤔 

Maybe leave the first one, but we don't need to repeat it, I think. Same thing with the rest of the repeated comments in this file; leave the first one in a section if you think it's needed, and remove the rest to keep from repeating too much.

Reply:
E.G. statements after the first removed in commit e7166428dbc22323b602db8d109028dd96564ef1